api/src/main/java/com/epam/pipeline/dao/pipeline/PipelineDao.java

```diff
@@ -253,6 +253,9 @@ private static void initPipeline(final ResultSet rs, final Pipeline pipeline) th
             if (!rs.wasNull()) {
                 pipeline.setParentFolderId(folderId);
             }
+            if (!rs.wasNull()) {
+                pipeline.setParent(new Folder(folderId));
```

Comment:
As far as I understand we have the same problem (that parent is not set during load, as result f.e. FolderApiService,loadProject method can't work correctly ) not only for pipeline but also at least for DataStorage and Folder, RunConfiguration. So, let's fix it as well